lib/src/main/java/vmath/Mat2.java

```diff
@@ -10,6 +10,27 @@ public Mat2() {
         };
     }
 
+    public Mat2(float[] initialValues) {
+        if (initialValues.length != 4) {
+            throw new IllegalArgumentException("Initial values array must have exactly 4 elements.");
+        }
+
+        for(int i = 0; i < 4; ++i)
+        {
+            if(Float.isNaN(initialValues[i]))
```

Comment:
Please add spaces on lines 18 and 20 in between "for"/"if" and the opening parentheses. The test file is also missing spaces between the numbers and the curly braces in several array initialization lists.

Reply:
Please, switch to the for-each loop like in other PRs for the NaN check.